### ISSUE TITLE
query parameter values are not escaped properly

### ISSUE THREAD

**Issue by alexandervanhecke** [ducaale/xh#54]
Hi,
I'm passing a query parameter of the form `geometry=POINT(1 2)` to a request : 

```zsh
ht post ".../api/...doStuff?exporttype=legacy&geometry=POINT(1 2)" <<< "[]"
```

please note the request is within double quotes

Using `ht` version `3.5.0`, my app receives `POINT(1` as the value of parameter geometry

**Comment by blyxxyz** [comment]
Good catch! I've tried to make it more reliable, see #55.

In the meantime you could work around the issue by using the GET parameter syntax:

```sh
xh post ".../api/...doStuff" exporttype==legacy geometry=='POINT(1 2)' <<< "[]"
```

**Comment by ducaale** [comment]
This is now fixed in `xh` version 0.8.0. Feel free to reopen this if you encounter a similar bug again.
